hudi-common/src/test/java/org/apache/hudi/common/model/TestHoodieCommitMetadata.java

```diff
@@ -34,6 +40,33 @@
  */
 public class TestHoodieCommitMetadata {
 
+  private static final List<String> EXPECTED_FIELD_NAMES = Arrays.asList(
+      "partitionToWriteStats", "compacted", "extraMetadata", "operationType");
+
+  public static void verifyMetadataFieldNames(
+      HoodieCommitMetadata commitMetadata, List<String> expectedFieldNameList)
+      throws IOException {
+    String serializedCommitMetadata = commitMetadata.toJsonString();
+    Iterator<String> fieldNameIterator = JsonUtils.getObjectMapper()
```

Comment:
You can do `CollectionUtils.toStream` for easier conversion

Reply:
Fixed.